src/app/component/Header/Header.component.js

```diff
@@ -33,6 +33,7 @@ import CSS from 'Util/CSS';
 import media from 'Util/Media';
 import { LOGO_MEDIA } from 'Util/Media/Media';
 
+import wishListShareButton from '../../../public/assets/images/global/shareWishList.svg';
```

Comment:
Can you please import SVG as an asset, not from public? :D

Reply:
Public folder is for files used in index.html / .phtml

Reply:
Maybe we have a similar icon in `_icons.scss`?

Reply:
arrow back + minicart can be a solution.